File: spotifyarch/src/main/java/com/timothy/spotifyarchitecture/retrofit/models/FeaturedPlaylists.java
package com.timothy.spotifyarchitecture.retrofit.models;

import android.os.Parcel;
import android.os.Parcelable;

public class FeaturedPlaylists {
    public String message;
    public Pager<PlaylistSimple> playlists;

    public FeaturedPlaylists() {
    }
}

File: spotifyarch/src/main/java/com/timothy/spotifyarchitecture/retrofit/models/FeaturedPlaylists.kt
package com.timothy.spotifyarchitecture.retrofit.models

import android.os.Parcel
import android.os.Parcelable

class FeaturedPlaylists {
    var message: String = ""
    var playlists: Pager<PlaylistSimple> = Pager<PlaylistSimple>()
}
